pom.xml

```diff
@@ -341,6 +341,49 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+
+                    <!--
+                      This insures that the windup-web-redirect directory exists.
+                      Without this, the CLI script may fail later on.
+                    -->
+                    <execution>
+                        <id>create-windup-web-redirect-directory</id>
+                        <phase>process-resources</phase>
+                        <configuration>
+                            <target>
+                                <mkdir dir="${project.build.directory}/${wildfly.directory}/windup-web-redirect"/>
```

Comment:
This is required as EAP 7.1 seems to have a stricter validator that requires the directory to be there before it will allow the CLI script to run.